### PR TITLE
Added UrlGuard.cs and updated ExceptionMiddleware.cs 

### PR DESCRIPTION
**Pull request by RedonAdemi**
Also put some comment about code

### CODE REVIEW

File: ReviveIT/Application/Common/Exceptions/UrlGuard.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace Application.Common.Exceptions
+{
+    public static class UrlGuard
+    {
+        public class UnsafeUrlException : Exception
+        {
+            public UnsafeUrlException(string message) : base(message) { }
+        }
+        // List of unsafe patterns or keywords, including single characters
```
Comment (donatgosalcii) [inline ReviveIT/Application/Common/Exceptions/UrlGuard.cs:15]: Remove comments from the code (refactor it).

File: ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs [hunk 2]
```diff
@@ -3,63 +3,94 @@
 
 namespace WebUI.MiddleWares
 {
+    // Middleware class for handling exceptions and unsafe URLs
```
Comment (donatgosalcii) [inline ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs:6]: Remove comments from this class too.

File: ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs [hunk 3]
```diff
@@ -3,63 +3,94 @@
...
+                }
+
+                // Continue processing the request
                 await _next(context);
             }
             catch (Exception exception)
             {
-                _logger.LogError(exception.Message, exception.StackTrace, exception.InnerException, exception.InnerException?.Message);
+                // Log the exception details for debugging purposes
+                _logger.LogError(
```
Comment (donatgosalcii) [inline ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs:41]: It was already made, but more readable like this, remove the comments.

**Review by donatgosalcii — CHANGES_REQUESTED** [review]
Pastroje kodin prej unescessary comments edhe check for white lines or spaces

File: ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs (hunk 3 above)
Comment (RedonAdemi) [reply ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs:32]: Donat recheck the code again - its changed now

**Review by donatgosalcii — APPROVED** [review]

**Review by NathanAmanuelG — APPROVED** [review]
The code is solid though I'd recommend you tone down refactoring your code so mush, less is more, either way good job!

**Pull request merged**